### PR TITLE
setup.py: flake8 is a superset of pep8 and pyflakes

### PR DESCRIPTION
**Pull request by cclauss**
__pep8__ was renamed to __pycodestyle__: https://pep8.readthedocs.io/en/release-1.7.x

### CODE REVIEW

**Comment by cclauss** [comment]
Closed in favor of onelogin/python3-saml#124

**Comment by pitbulk** [comment]
I think it makes sense to also apply the code fixes and the flake8 on the python-saml project.
Can you send a PR? I think you already worked on it for python-saml before prepare the PR for python3-saml

File: src/onelogin/saml2/utils.py [hunk 1]
```diff
@@ -434,7 +434,7 @@ def generate_unique_id():
 
     @staticmethod
     def parse_time_to_SAML(time):
-        """
+        r"""
```
Comment (pitbulk) [inline src/onelogin/saml2/utils.py]: That r in the comment is because it contains a regular expression?

Comment (pitbulk) [reply src/onelogin/saml2/utils.py]: Is ok then to add an r in the description of a method?

Comment (cclauss) [reply src/onelogin/saml2/utils.py]: src/onelogin/saml2/utils.py:446:-151: W605 invalid escape sequence '\.'
src/onelogin/saml2/utils.py:461:-228: W605 invalid escape sequence '\.'

**Pull request merged**